test/lsp-clients-test.el

```diff
@@ -75,4 +75,12 @@
   (let ((major-mode 'js-mode))
     (should (not (lsp-clients-flow-activate-p (concat test-location "fixtures/SampleTypeScriptProject/src/sample.ts") nil)))))
 
+(ert-deftest lsp-typescript-javascript-activates-based-on-file-extension ()
+  (should (lsp-typescript-javascript-tsx-jsx-activate-p "abc.js"))
```

Comment:
can you add the following tests `a1.ts` and `a1.d.ts`?

Reply:
Done